frontend/pages/hosts/details/DeviceUserPage/components/InfoButton/InfoButton.tsx

```diff
@@ -0,0 +1,25 @@
+import Button from "components/buttons/Button";
+import Icon from "components/Icon";
+import React from "react";
+
+const baseClass = "info-button";
+
+interface IInfoButton {
+  toggleInfoModal: () => void;
```

Comment:
Nit: if the point is for this to be a component we can use whenever we want a small button with the info icon, I'd just make this `onClick` rather than `toggleInfoModal`, since there may be use cases where it does something other than opening a modal.

Reply:
agree